about.qmd

```diff
@@ -73,7 +75,8 @@ If you would like to see a blog post on a particular topic, submit an issue on o
 - [X] [Cell typing: what we've found to work](posts/cell-typing-basics/index.qmd)
 - [X] [On the use of marker genes](posts/on-cell-typing-with-marker-genes/index.qmd)
 - [ ] Hierarchical ("plinko") cell typing
-- [ ] Cell typing with smoothed marker genes
+- [X] [Cell typing with smoothed marker genes](posts/marker-gene-smoothing/index.qmd)
+- [X] [Creating reference profiles](/posts/hybrid-reference-profiles/index.qmd)
```

Comment:
Might be good to move this one on reference profiles above the Hierarchical cell typing. 
Btw, the spatial information one seems to be relevant to cohort usage, not sure if it's discussed else where.

Reply:
Reference profiles now second. The spatial information one is a planned post but I don't believe we've written anything - agree it would be good to discuss cohorts for that one